python/sglang/srt/layers/attention/flashattention_backend.py

```diff
@@ -254,7 +397,26 @@ def forward_extend(
             else (-1, -1)
         )
 
-        page_table = metadata.page_table
+        # Check if we should use local attention
+        use_local_attn = (
+            self.use_local_attention and metadata.local_attn_metadata is not None
```

Comment:
Out of curiosity, is this attention backend for all layers or partial layers only so `use_local_attention` is not applied for the full attention layers? (wondering if we need layer id to decide this together if the backend is shared by the local attention layer and full attention layers) Thank you!